src/gc/softwarewritewatch.h

```diff
@@ -11,6 +11,11 @@
 #ifdef FEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP
 #ifndef DACCESS_COMPILE
 
+#ifndef GC_PAGE_SIZE
+#define GC_PAGE_SIZE 0x1000
+#endif
```

Comment:
This is defined in gcpriv.h in the 64K PR, but I could not get gcpriv.h included in this file.